cycode/cli/code_scanner.py

```diff
@@ -336,7 +337,19 @@ def scan_disk_files(context: click.Context, path: str, files_to_scan: List[str])
         if not content:
             continue
 
-        documents.append(Document(file, content, is_git_diff))
+        file_name = file
+
+        if _is_iac(scan_type) and _is_tfplan_json_file(file, content):
```

Comment:
scan_disk_files is relevant only for scan type `path`, is it intended? we don't  want to support it also in `repository` and `commit_range`?

Reply:
why we do the logic here and not in the server, its not something that should be done also as part of repository/pr scan?

Reply:
@MichalBor No, tf plan is not something you are pushing to a repo.
It contains secrets and addition data the injected during build process. 
hence, it's not relevant for repo scan and not implemented in BE.

Reply:
Running terraform plan is only on CI/CD process,  the use-case is that the customer will run it on his build system